overlays/low-resource/node-exporter/delete-Daemonset.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
```

Comment:
what does this delete do?

Reply:
It deletes the node-exporter daemonset when someone uses the low-resource kustomize overlay.